invent-code-1/main.cpp

```diff
@@ -7,6 +7,10 @@ int main() {
   string city;
 
   // Step 1. Ask the user for their name and their city.
-
+cout << "what is your name" << " ";
+cin >> name;
+cout << "which city are you from" << " ";
+cin >> city;
   // Step 2. Print the greeting back to the user.
+cout << "Nice to meet you" <<" " << name <<" " << "from" <<" " << city <<". Have a great day!"; 
```

Comment:
You can just write "Nice to meet you " with the space attached to the first string. This can simplify the code. Same for " from ", with the spaces in the same string.